flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -1686,6 +1686,7 @@
                             <LOCK_NODE_ID>LOCK_OVHD_EVAC_COMMAND</LOCK_NODE_ID>
 
                             <TOGGLE_SIMVAR>L:A32NX_EVAC_COMMAND_TOGGLE</TOGGLE_SIMVAR>
+                            <DOWN_CODE>(L:A32NX_EVAC_COMMAND_TOGGLE) !</DOWN_CODE>
```

Comment:
```suggestion
                            <INVERTED/>
```

Reply:
The `INVERTED` parameter only affects the lights on the button. I have created a new parameter called `INVERTED_ANIMATION` for this.